gap/semigroups/semitrans.gi

```diff
@@ -735,6 +735,47 @@ function(S)
   return t;
 end);
 
+InstallMethod(WreathProduct, "for a monoid and a permutation group",
+[IsTransformationSemigroup, IsPermGroup],
+function(S, G)
+  local maps, gensS, next, reps, n, i, g, x, m;
+  if not IsMonoidAsSemigroup(S) then
+    ErrorNoReturn("S should be a monoid");
```

Comment:
This error should be in the format: `Semigroups: WreathProduct: usage..`
please see other error messages in the package for the correct format.